docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  database:
+    image: 'postgres:latest'
```

Comment:
Let's fix the version to `16` instad of using `latest`, we want to create the same container regardless of what the latest version is (we can always update it).